cfme/tests/cloud_infra_common/test_power_control_rest.py

```diff
@@ -53,7 +53,8 @@ def _state_changed():
         vm.reload()
         return vm.power_state == state
     wait_for(_state_changed, num_sec=num_sec, delay=45, silent_failure=True,
-        message="Wait for VM state `{}` (current state: {})".format(state, vm.power_state))
+        message="Wait for VM state `{}` (current state: {})".format(state, vm.power_state),
+        fail_func=vm_obj.refresh_relationships)
```

Comment:
**REQUIRED**: Is it OK to refresh every 45 seconds?

Reply:
It's a good point. Honestly, it does not seem to be wrong to me. But I might extend it to like 60 or 90 seconds maybe. I would not go for more though. This would make the test case unnecessarily long.

Reply:
Sure :) - happy!